src/lib/common/limits.h

```diff
@@ -94,6 +95,7 @@
 */
 #define  LOCAL_IP_V6  "::"
 #define  LOCAL_IP_V4  "0.0.0.0"
+#define  MAX_LEN_IP   64
```

Comment:
For IPv4 the calculation is perfect... What about a similar calculation for IPv6?